.travis.yml

```diff
@@ -6,7 +6,7 @@ rvm:
   - 2.2.6
   - jruby-9.0.5.0
 gemfile:
-  - gemfiles/ar50.gemfile
+  - gemfiles/ar51.gemfile
```

Comment:
the main issue is that the signature of `type_to_sql` [changed to accept keyword arguments](https://github.com/rails/rails/commit/ae39b1a0#diff-a0775e1ec64264dc76a8ee71a5211ae3) and i'm not aware of a backward-compatible _and_ elegant approach to support both. i wouldn't want to use ruby version checks just to have cross-version support.

separate branches seem to be the way to go.